Primo.js

```diff
@@ -442,6 +447,7 @@ function importPNX(text) {
 			item.attachments.push({url: ZU.xpathText(doc, '//display/lds03'), title: "HOLLIS Permalink", snapshot: false});		
```

Comment:
Actually, it would be nice to scrape the permalink in general, which normally means make another call to some url e.g. `http://primo.bib.uni-mannheim.de/primo_library/libweb/action/permalink.do?docId=MAN_ALMA21146476460002561&vid=MAN_UB&fn=permalink` and then extract that link.